src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+
+  --color: #00ACDC;
 }
+
+
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+
+}
+
+.nav {
+  position: sticky;
+  top: 0;
```

Comment:
1. unnecessary, if you want to make your header sticky please add position sticky to the whole header
2. your element is on top, you don't need `top: 0`, it does nothing here